spread.yaml

```diff
@@ -408,7 +416,7 @@ suites:
 
     tests/unit/:
         summary: Suite to run unit tests (non-go and different go runtimes)
-        systems: [-ubuntu-core-16-*, -fedora-*]
+        systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*]
```

Comment:
We can run unit tests on openSUSE but as explained before this PR just adds build support for openSUSE and doesn't enable any spread tests (in real we have to enable at least a single one tests/main/help to get the build running). Enabling tests will happen with later PRs step by step.